src/components/RenderList.vue

```diff
@@ -1,34 +1,35 @@
 <template>
   <div>
-      <div v-for="item in items" :key="item.id">
+      <div v-for = "(item, index) in items" :key = "item.id">
```

Comment:
not the right way to use spaces
in HTML use `v-for=""`

Reply:
✔️